assets/airlines/emb.json

```diff
@@ -11,10 +11,8 @@
     },
     "fleets": {
         "default": [
-            ["E50P", 5],
```

Comment:
Why have the `E50P`, `E55P`, `E545` aircraft types been removed? They should be readded.